### PR TITLE
Very simple sqlite example - addbk.py

### PR DESCRIPTION
**Pull request by xircon**
Just thought I would share.

### CODE REVIEW

**Comment by Pcothren** [comment]
Do we need the pyaoutgui command?
Also when I open the address data base after using the example there isnt any data in it?

**Comment by xircon** [comment]
Oops, thought I deleted that line sorry.  Definitely works here:
```
sqlite3 addresses.db
SQLite version 3.34.0 2020-12-01 16:14:00
Enter ".help" for usage hints.
sqlite> select * from address_data;
1|Mickey|Mouse|Disney World |Florida|||TEST
2|1|1|1|1|1|1|1
```

**Comment by Pcothren** [comment]
okay nice! Although its self explanatory app, we need some reference text, Ill approve and you can add it or if you merger all go and add it after things such as "this app features an SQL Lite data base with saving ability" after the data entru the data base will be created in the location of the .py file" something along those lines. Thanks for your addition, we will get it featured on the website asap!

**Review by Pcothren — APPROVED** [review]

File: addbk.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    # Write to table:
+    con = sqlite3.connect('addresses.db')
+    cursor = con.cursor()
+    sqlite_insert_query = """INSERT INTO address_data
+                          ( fname, sname, add1, add2, add3, city, pcode) 
+                          VALUES (?, ?, ?, ?, ?, ?, ?);"""
+    cursor.execute(sqlite_insert_query, (tFN, tSN, tA1, tA2, tA3, tCI, tPC))
+    con.commit()
+    con.close()
+    pyautogui.hotkey('tab')
```
[the remark this answers is not in the document]
Comment (xircon) [reply addbk.py:76]: Yes as I said, was a failed experiment to focus the input boxes and I thought I deleted it.  Will do the reference text tomorrow, been a long day at work...

[the remark this answers is not in the document]
Comment (Pcothren) [reply addbk.py:76]: no problem, great example here, thanks!

**Review by Pcothren — APPROVED** [review]

**Comment by xircon** [comment]
I have improved and commented the code, I don't know git, what do I do?

**Review by Pcothren — DISMISSED** [review]

**Review by Pcothren — APPROVED** [review]

**Pull request merged**